### ISSUE TITLE
sir, sorry to disturb you but i can't understand the code between line 78 to line 97, for nonetype has no attribute 'value' but you use the value of s, can you make an explanation on it?

### ISSUE THREAD

**Issue by schuy1er** [M-Elfeki/GDPP#3]
sir, sorry to disturb you but i can't understand the code between line 78 to line 97, for nonetype has no attribute 'value' but you use the value of s, can you make an explanation on it?

**Comment by HelenMao** [comment]
I have the same problem, and I modify it directly as len(mode_counter).

**Comment by Johnson-yue** [comment]
@HelenMao it has worked for modifing it as len(mode_counter)?

**Comment by HelenMao** [comment]
@Johnson-yue  Right, I implement it in my project and it has worked. You can try it.
